src/lavatory/utils/artifactory.py

```diff
@@ -1,100 +1,92 @@
+import base64
 import datetime
 import json
 import logging
 
-import requests
+import party
 
-LOG = logging.getLogger(__name__)
-
-
-class _Http(object):
-
-    def __init__(self, baseurl, user, passwd):
-        self.baseurl = baseurl
-        self.apiurl = baseurl + "/api"
-        self.auth = (user, passwd)
-
-    def get(self, endpoint):
-        response = requests.get(self.apiurl+endpoint, auth=self.auth)
-
-        if response.ok:
-            return response.json()
-        else:
-            response.raise_for_status()
-
-    def post(self, endpoint, payload):
-        response = requests.post(self.apiurl+endpoint, auth=self.auth, data=payload)
-
-        if response.ok:
-            return response.json()
-        else:
-            response.raise_for_status()
+from ..credentials import load_credentials
 
-    def delete(self, path):
-        LOG.info("DELETE", path)
-        return
+LOG = logging.getLogger(__name__)
 
-        """
-        response = requests.delete(baseurl + "/" + path)
+CREDENTIALS = load_credentials()
```

Comment:
I think this should happen inside the class when it is initialized.

Reply:
Moved inside the __init__.